cli/src/commands/download.ts

```diff
@@ -0,0 +1,91 @@
+import { CLIError } from '@oclif/errors';
+import * as path from 'path';
+import * as tar from 'tar';
+import * as request from 'request';
+import { spawn } from '@binaris/utils-subprocess';
+import Command from '../utils/command';
+import { Project } from '../utils/helpers';
+
+export default class Download extends Command {
+  public static description = 'download application';
+  public static examples = [
+    `$ ${Command.cliBinName} download 123456`,
+  ];
+
+  public static args = [
+    {
+      name: 'ID',
+      required: true,
+      description: 'application id',
```

Comment:
AFAIK other commands default to the application in the current directory.  It _would_ make sense to download my app, e.g. if I want to make sure what's running.
Can we make it optional _when_ in the directory of an application?